src/i18n/locales/en/translation.json

```diff
@@ -796,10 +795,26 @@
       "title": "Sign",
       "content": "This website asks you to sign:"
     },
-    "confirm_get_address": {
-      "title": "Get Address",
-      "heading": "This website asks you to read:",
-      "content": "Your Bitcoin receive address"
+    "confirm_sign_pset": {
+      "warning": "⚠️ Signing a PSET may result in loosing your funds or Ordinals. Sign only if you understand what you're doing and trust this transaction",
+      "title": "Sign PSET",
+      "allow_sign": "This website asks you to sign a Partially Signed Elements (Liquid) Transaction:",
```

Comment:
Simplify wording, maybe link to a help page (learn more button)

Reply:
@bumi @reneaaron changed to 
![image](https://github.com/getAlby/lightning-browser-extension/assets/33993199/9e50bac6-6dc5-41a6-99be-befb972965f8)